pkg/sources/git/git.go

```diff
@@ -791,7 +790,7 @@ func stripPassword(u string) (string, error) {
 
 	repoURL, err := url.Parse(u)
 	if err != nil {
-		return "", errors.WrapPrefix(err, "repo remote cannot be sanitized as URI", 0)
+		return "", fmt.Errorf("repo remote cannot be sniffed as URI: %w", err)
```

Comment:
**Nit:** I had to re-read this error message a couple of times and look at the context before it clicked what it is conveying. What about `repo remote is not a URI`?